backend/grant-service/src/main/kotlin/grant/rpc/GrantController.kt

```diff
@@ -170,6 +191,30 @@ class GrantController(
             ok(settings.browse(db, ctx.securityPrincipal.toActor(), request.normalize()))
         }
 
+        implement(Grants.findAffiliations) {
+            val user = UserDescriptions.getPrincipal.call(
+                GetPrincipalRequest(request.username),
+                serviceClient
+            ).orThrow()
+            user as Person
```

Comment:
This crashes with an internal server error if the user is not a `Person` should instead throw a `NotFound` exception.